MvvmCross-Forms/MvvmCross.Forms/Views/NavigationHierarchyHelper.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Linq;
+using Xamarin.Forms;
+using System.Text;
+
+namespace MvvmCross.Forms.Views
+{
+    public static class MvxNavigationHierarchyHelper
+    {
+        public static bool PageMatchesViewModel(this Page page, Type viewModelType)
```

Comment:
This is not a HierarchyHelper but a normal helper. Please add another class `MvxFormsExtensions` and move it to that.